src/wizard/boards.ts

```diff
@@ -1,22 +1,26 @@
 import { html } from "lit";
 import { SupportedBoards } from "../api/boards";
 
-export const boardSelectOptions = (boards: SupportedBoards | undefined) =>
+export const boardSelectOptions = (
+  boards: SupportedBoards | undefined,
+  defaultBoard: string | null = null
+) =>
   boards &&
-  html`
-    <optgroup label="ESP32">
-      ${Object.keys(boards.esp32).map(
-        (key) => html`<option value="${key}">${boards.esp32[key]}</option>`
+  Object.values(boards).map((group) => {
+    const hasDefault =
+      defaultBoard && Object.keys(group.items).includes(defaultBoard);
+    return html`
+      ${hasDefault &&
+      html`<option value="${defaultBoard}" selected>
+        ${group.items[defaultBoard]}
+      </option>`}
+      ${hasDefault && html`<option disabled>------</option>`}
+      ${group.title && html`<optgroup label="${group.title}"></optgroup>`}
+      ${Object.keys(group.items).map(
+        (key) =>
+          key !== defaultBoard &&
+          html`<option value="${key}">${group.items[key]}</option>`
```

Comment:
Lit is not React and does render booleans.

![CleanShot 2022-12-11 at 22 03 41](https://user-images.githubusercontent.com/1444314/206951881-adfbbd29-880a-4245-bf5d-2e956dbfa51e.png)


```suggestion
          key === defaultBoard ?
            "" :
            html`<option value="${key}">${group.items[key]}</option>`
```